packages/molly/src/routes/chat/components/chatFeed.svelte

```diff
@@ -0,0 +1,64 @@
+<script lang="ts">
+	import { onMount } from 'svelte';
+	import Logo from './chatLogo.svelte';
+	import ChatMessage from './chatMessage.svelte';
+
+	export let data: Array<Object>;
+
+	onMount(() => {
+		//Scroll to the bottom of the chat window
+		const chatWindow: HTMLElement | null = document.querySelector('.chat-content');
```

Comment:
OpenAI also complained about this, but using Svelte's own `this` selector is a really nice feature. Eliminates the need to querySelect anything (we also spoke about this in DMs, but I thought I'd add it as a comment as well just in case)

Reply:
I've pushed a change to this